### PR TITLE
Add NTAP to supporting funders list (they are on the synapse footer already).

### PR DESCRIPTION
**Pull request by allaway**
This pull request has been automatically generated by prose.io.

### CODE REVIEW

**Comment by kelshmo** [comment]
Yay, prose!

File: _articles/faq.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
-title: "FAQ"
+title: FAQ
 layout: article
 excerpt: What is Synapse?
 order: 4
+published: true
```
Comment (kelshmo) [inline _articles/faq.md:6]: @allaway did prose add this to the yaml? I don't think this will be interpreted in the build.

**Review by kelshmo — CHANGES_REQUESTED** [review]

**Comment by allaway** [comment]
Hey Kelsey - yes, it did. I PRed anyway because I thought that the branches
were set up to build before merging, to see if this would work, but i guess
the branches dont' automatically build?

Anyways, I'll check to see if there is a prose-side configuration I can
change to fix this.

To be honest, the purpose of this PR was more to test out prose, I just
happened to find this change as something I could make while testing :)

**Comment by kelshmo** [comment]
The GH Actions build is currently not set-up however, I tested this locally with Jekyll and it is building fine. 

The `published:true` tag is being ignored. I will probably just remove it to not have it as an artifact in the code that is currently not functional - do you want to push these changes to the live site?

**Comment by allaway** [comment]
Yep! I removed the `published: true tag`, feel free to merge if you are happy with it! 

I took a look around and found [this issue](https://github.com/prose/prose/issues/1097), it doesn't look like we're the only one experiencing this. 

I'll take a look at some other editors to see how easy they are to use and whether they do anything unexpected like this. Prose is nice because it obfuscates a lot of git jargon away...

**Review by kelshmo — APPROVED** [review]

**Comment by hmua** [comment]
> Yep! I removed the `published: true tag`, feel free to merge if you are happy with it!
> 
> I took a look around and found [this issue](https://github.com/prose/prose/issues/1097), it doesn't look like we're the only one experiencing this.
> 
> I'll take a look at some other editors to see how easy they are to use and whether they do anything unexpected like this. Prose is nice because it obfuscates a lot of git jargon away...

Could you please help to point out how to do this?

**Pull request merged**